src/test/resources/com/puppycrawl/tools/checkstyle/filters/strategy/FileTabCharacter/Test.java

```diff
@@ -0,0 +1,26 @@
+package Checker.FileTabCharacter;
+
+public class Test {
+    // Long line ----------------------------------------------------------------
+    // Contains a tab ->	<- //warn
+    // Contains trailing whitespace ->
```

Comment:
Looks like you don't need trailing spaces in this input file. FileTabCharacter doesn't check them

Reply:
done